types/src/primary.rs

```diff
@@ -714,6 +712,13 @@ pub enum PrimaryWorkerMessage {
     DeleteBatches(Vec<BatchDigest>),
 }
 
+/// Used by the primary to request that the worker sync the target missing batches.
+#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
Do we need `Clone`?

Reply:
Yes we do, because of https://github.com/MystenLabs/narwhal/blob/aschran/nwproto/network/src/traits.rs#L10